src/templates/components/affiliate-multiple-event-component-city.njk

```diff
@@ -0,0 +1,39 @@
+<!-- MULTIPLE EVENTS -->
+<table role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" class="emailwrapto100pc" width="600">
+    <tbody>
+        <tr>
+            <td style="font-family: Helvetica, Arial, sans-serif; font-size:16px; color:#2a2a2a;" bgcolor="#f2f2f2">
+                <table role="presentation" bgcolor="#b2cce5" width="100%" border="1" cellspacing="0" cellpadding="0" style="background-color:#b2cce5; border-radius: 3px; border-collapse: separate;" align="right" class="footer-right">
```

Comment:
what is the `class="footer-right"` doing here? Is this something you added, or was it just there when you copied the table?

Reply:
looks like we have a `bgcolor="#b2cce5"` and `style="background-color:#b2cce5"`. This is redundant and should remove one. I would remove the bgcolor since we cannot update that through media queries.